client/blocks/location-search/prediction.jsx

```diff
@@ -0,0 +1,38 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+
+export default class Prediction extends Component {
+	static propTypes = {
+		onPredictionClick: PropTypes.func,
+		prediction: PropTypes.object.isRequired,
+	};
+
+	handlePredictionClick = () => {
+		const { onPredictionClick } = this.props;
+		if ( typeof onPredictionClick === 'function' ) {
+			onPredictionClick( this.props.prediction );
+		}
```

Comment:
Not mandatory at all, but we could have defined a default prop set to `noop` to simplify this code.